source/globalCommands.py

```diff
@@ -4259,6 +4259,12 @@ def _enableScreenCurtain(doEnable: bool = True):
 					)
 				)
 			else:
+				from contentRecog.recogUi import RecogResultNVDAObject
+				if isinstance(api.getFocusObject(), RecogResultNVDAObject):

```

Comment:
Could this condition be modified so that screen curtain is disallowed only if auto refresh is enabled?

Or was there any other condition preventing screen curtain to be used while in OCR result?